paramak/parametric_shapes/rotate_spline_shape.py

```diff
@@ -10,67 +10,26 @@ class RotateSplineShape(Shape):
     """Rotates a 3d CadQuery solid from points connected with splines
 
     Args:
-        points (list of tuples each containing X (float), Z (float)): A list of
-            XZ coordinates connected by spline connections. For example [(2.,
-            1.), (2., 2.), (1., 2.), (1., 1.)].
-        name (str): The legend name used when exporting a html graph of the
-            shape.
-        color (RGB or RGBA - sequences of 3 or 4 floats, respectively, each in the range 0-1):
-            The color to use when exporting as html graphs or png images.
-        material_tag (str): The material name to use when exporting the
-            neutronics description.
-        stp_filename (str): The filename used when saving stp files as part of a
-            reactor.
-        azimuth_placement_angle (float or iterable of floats): the angle or
-            angles to use when rotating the shape on the azimuthal axis.
-        rotation_angle (float): The rotation_angle to use when revoling the
-            solid (degrees).
-        cut (CadQuery object): An optional cadquery object to perform a boolean
-            cut with this object.
-
-    Returns:
-        a paramak shape object: a Shape object that has generic functionality
+        rotation_angle (float, optional): The rotation_angle to use when
+            revolving the solid (degrees). Defaults to 360.0.
+        solid (cadquery.Workplane, optional): [description]. Defaults to None.
```

Comment:
@Shimwell any idea ?

Reply:
I think we can remove the entire return statement

Reply:
I vote remove the return statement

Reply:
The return statement has been removed already, the comment was to know what to put instead of [description] in the solid argument description